From: Felix Fietkau <nbd@openwrt.org>
Date: Tue, 3 Nov 2015 12:00:04 +0000
Subject: tools/bison: update to 3.0.4

Update bison to 3.0.4.
Refresh patches.
Drop the unnecessary PKG_CAT definition.

Signed-off-by: Hannu Nyman <hannu.nyman@iki.fi>

SVN-Revision: 47370

diff --git a/tools/bison/Makefile b/tools/bison/Makefile
index 256512116f9f7b8f90f2dcce24846286e5c50025..b8ad32c561f2d95ade66465cefb5eeaec74f64c2 100644
--- a/tools/bison/Makefile
+++ b/tools/bison/Makefile
@@ -1,5 +1,5 @@
 # 
-# Copyright (C) 2008-2013 OpenWrt.org
+# Copyright (C) 2008-2015 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
@@ -7,12 +7,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=bison
-PKG_VERSION:=3.0.2
+PKG_VERSION:=3.0.4
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@GNU/$(PKG_NAME)
-PKG_MD5SUM:=146be9ff9fbd27497f0bf2286a5a2082
-PKG_CAT:=xzcat
+PKG_MD5SUM:=c342201de104cc9ce0a21e0ad10d4021
 
 HOST_BUILD_PARALLEL:=1
 
diff --git a/tools/bison/patches/100-fix-gets-removal.patch b/tools/bison/patches/100-fix-gets-removal.patch
index b9c878a2771ba0ecb38342fcb8aad4d650f7f107..20b18cf2e7cd722622fced5b0e7703807bef38ef 100644
--- a/tools/bison/patches/100-fix-gets-removal.patch
+++ b/tools/bison/patches/100-fix-gets-removal.patch
@@ -1,6 +1,6 @@
 --- a/lib/stdio.in.h
 +++ b/lib/stdio.in.h
-@@ -714,13 +714,6 @@
+@@ -719,13 +719,6 @@
  # endif
  #endif
  
